internal/alloycli/cmd_run.go

```diff
@@ -197,9 +197,7 @@ func (fr *alloyRun) Run(configPath string) error {
 
 	// Set the memory limit, this will honor GOMEMLIMIT if set
 	// If there is a cgroup will follow that
-	if fr.minStability.Permits(featuregate.StabilityPublicPreview) {
```

Comment:
Want to remove the `Permits` function? 😂 
It could be useful though, just that I don't think it's used now so YAGNI.